h/conftest.py

```diff
@@ -91,6 +92,8 @@ def db_session(request, settings):
     engine = db.make_engine(settings)
     db.bind_engine(engine, should_create=True, should_drop=True)
 
+    api_db.use_session(db.Session)
```

Comment:
No, `make_engine()` doesn't call `use_session()`, the `includeme()` does. I could refactor `h/db.py` and `h/api/db.py` to make this line unnecessary here though.